main.go

```diff
@@ -66,7 +66,7 @@ func main() {
 		azcopyJobPlanFolder = path.Join(azcopyAppPathFolder, "plans")
 	}
 	if err := os.Mkdir(azcopyJobPlanFolder, os.ModeDir|os.ModePerm); err != nil && !os.IsExist(err) {
-		common.PanicIfErr(err)
+		log.Fatalf("Problem making .azcopy directory. Try setting AZCOPY_PLAN_FILE_LOCATION env variable. %v", err)
```

Comment:
It seems to me the name of this variable is actually `AZCOPY_JOB_PLAN_LOCATION` and this error message, while meant to be helpful, is somewhat misleading.